data-service/src/main/java/com/amazonaws/blox/dataservice/model/Cluster.java

```diff
@@ -29,12 +29,12 @@
 // required for mapstruct
 @NoArgsConstructor
 public class Cluster {
-  private static final String UNDERSCORE = "_";
+  private static final String JOINER = "_";
```

Comment:
Still underscore?

Reply:
maybe to use `DELIMITER` as the var name?

Reply:
Yeah, that's the word... Fixed.